<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigurationOutputStreaming</class>
 <widget class="QWidget" name="ConfigurationOutputStreaming">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>858</width>
    <height>482</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="minimumSize">
        <size>
         <width>850</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaContainer">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>836</width>
          <height>583</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <property name="spacing">
            <number>16</number>
           </property>
           <item row="2" column="0">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="serviceSettingsLayout"/>
           </item>
           <item row="1" column="0">
            <widget class="QWidget" name="configurationWidget" native="true">
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QGroupBox" name="groupBoxGeneralSettings">
                <property name="title">
                 <string>General settings</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_3">
                 <property name="spacing">
                  <number>12</number>
                 </property>
                 <item row="0" column="1">
                  <widget class="QComboBox" name="comboReductions"/>
                 </item>
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_5">
                   <property name="minimumSize">
                    <size>
                     <width>200</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>200</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Output size</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="labelFPS">
                   <property name="text">
                    <string>FPS</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QLabel" name="inputFpsLabel">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="text">
                    <string notr="true">input fps</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="2">
                  <widget class="QLabel" name="labelFPSDesc">
                   <property name="text">
                    <string>Output framerate is defined by the input framerate</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBoxEncoderSettings">
                <property name="title">
                 <string>Encoder settings</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_4">
                 <property name="spacing">
                  <number>12</number>
                 </property>
                 <item row="0" column="0">
                  <widget class="QLabel" name="labelCodec">
                   <property name="text">
                    <string>Encoder</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QComboBox" name="encoderBox"/>
                 </item>
                 <item row="0" column="2">
                  <widget class="QLabel" name="labelPreset">
                   <property name="text">
                    <string>Preset</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="3">
                  <widget class="QComboBox" name="presetBox"/>
                 </item>
                 <item row="2" column="0">
                  <widget class="QLabel" name="label_4">
                   <property name="text">
                    <string>Profile</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1">
                  <widget class="QComboBox" name="profileBox"/>
                 </item>
                 <item row="6" column="0">
                  <widget class="QLabel" name="qualityBalanceLabel">
                   <property name="text">
                    <string>Quality balance</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="3">
                  <widget class="QLabel" name="levelLabel">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="text">
                    <string notr="true">3</string>
                   </property>
                  </widget>
                 </item>
                 <item row="6" column="1" colspan="3">
                  <layout class="QHBoxLayout" name="horizontalLayout_5">
                   <item>
                    <widget class="QLabel" name="minQualityLabel">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="text">
                      <string>higher compression</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QSlider" name="qualityBalanceSlider">
                     <property name="minimumSize">
                      <size>
                       <width>0</width>
                       <height>0</height>
                      </size>
                     </property>
                     <property name="maximum">
                      <number>32</number>
                     </property>
                     <property name="value">
                      <number>20</number>
                     </property>
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="maxQualityLabel">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="text">
                      <string>better quality</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item row="4" column="2">
                  <widget class="QLabel" name="bitrateLabel">
                   <property name="text">
                    <string>Bitrate</string>
                   </property>
                  </widget>
                 </item>
                 <item row="4" column="1">
                  <widget class="QComboBox" name="comboBitrateModes">
                   <property name="focusPolicy">
                    <enum>Qt::TabFocus</enum>
                   </property>
                  </widget>
                 </item>
                 <item row="5" column="0" colspan="2">
                  <widget class="QCheckBox" name="useAutomaticBitrateBox">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Automatic bitrate adjustment to network</string>
                   </property>
                   <property name="checked">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="5" column="2">
                  <widget class="QLabel" name="minBitrateLabel">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Min bitrate</string>
                   </property>
                  </widget>
                 </item>
                 <item row="5" column="3">
                  <widget class="QSpinBox" name="minBitrateBox">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="suffix">
                    <string> kbits/s</string>
                   </property>
                  </widget>
                 </item>
                 <item row="8" column="0" colspan="4">
                  <widget class="QGroupBox" name="advancedModeBox">
                   <property name="title">
                    <string>Advanced parameters</string>
                   </property>
                   <property name="checkable">
                    <bool>true</bool>
                   </property>
                   <layout class="QHBoxLayout" name="horizontalLayout">
                    <property name="rightMargin">
                     <number>0</number>
                    </property>
                    <item>
                     <widget class="QWidget" name="widget_2" native="true">
                      <layout class="QGridLayout" name="gridLayout_5">
                       <property name="leftMargin">
                        <number>0</number>
                       </property>
                       <property name="topMargin">
                        <number>0</number>
                       </property>
                       <property name="rightMargin">
                        <number>0</number>
                       </property>
                       <property name="bottomMargin">
                        <number>0</number>
                       </property>
                       <property name="horizontalSpacing">
                        <number>6</number>
                       </property>
                       <item row="1" column="1">
                        <widget class="QSpinBox" name="bufferSizeBox">
                         <property name="minimumSize">
                          <size>
                           <width>0</width>
                           <height>0</height>
                          </size>
                         </property>
                         <property name="suffix">
                          <string> kbits</string>
                         </property>
                         <property name="maximum">
                          <number>1000000</number>
                         </property>
                        </widget>
                       </item>
                       <item row="2" column="0" colspan="2">
                        <widget class="QCheckBox" name="cbrPaddingBox">
                         <property name="minimumSize">
                          <size>
                           <width>0</width>
                           <height>0</height>
                          </size>
                         </property>
                         <property name="text">
                          <string>Enable CBR padding</string>
                         </property>
                         <property name="checked">
                          <bool>true</bool>
                         </property>
                        </widget>
                       </item>
                       <item row="1" column="0">
                        <widget class="QCheckBox" name="useCustomBufferSizeBox">
                         <property name="minimumSize">
                          <size>
                           <width>0</width>
                           <height>0</height>
                          </size>
                         </property>
                         <property name="text">
                          <string>Use custom buffer size</string>
                         </property>
                         <property name="checked">
                          <bool>true</bool>
                         </property>
                        </widget>
                       </item>
                       <item row="0" column="0">
                        <widget class="QLabel" name="label_3">
                         <property name="minimumSize">
                          <size>
                           <width>200</width>
                           <height>0</height>
                          </size>
                         </property>
                         <property name="maximumSize">
                          <size>
                           <width>200</width>
                           <height>16777215</height>
                          </size>
                         </property>
                         <property name="text">
                          <string>GOP</string>
                         </property>
                        </widget>
                       </item>
                       <item row="0" column="1">
                        <widget class="QSpinBox" name="spinGOP">
                         <property name="minimumSize">
                          <size>
                           <width>0</width>
                           <height>0</height>
                          </size>
                         </property>
                         <property name="focusPolicy">
                          <enum>Qt::StrongFocus</enum>
                         </property>
                         <property name="specialValueText">
                          <string>auto</string>
                         </property>
                         <property name="suffix">
                          <string> frames</string>
                         </property>
                         <property name="minimum">
                          <number>-1</number>
                         </property>
                         <property name="maximum">
                          <number>999999999</number>
                         </property>
                         <property name="value">
                          <number>-1</number>
                         </property>
                        </widget>
                       </item>
                       <item row="3" column="0" colspan="2">
                        <widget class="QCheckBox" name="lowLatencyBox">
                         <property name="minimumSize">
                          <size>
                           <width>0</width>
                           <height>0</height>
                          </size>
                         </property>
                         <property name="text">
                          <string>Low latency</string>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item row="4" column="0">
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>Bitrate mode</string>
                   </property>
                  </widget>
                 </item>
                 <item row="4" column="3">
                  <widget class="QSpinBox" name="spinBitRate">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="focusPolicy">
                    <enum>Qt::StrongFocus</enum>
                   </property>
                   <property name="accelerated">
                    <bool>true</bool>
                   </property>
                   <property name="suffix">
                    <string> kbits/s</string>
                   </property>
                   <property name="minimum">
                    <number>1</number>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="2">
                  <widget class="QLabel" name="label_9">
                   <property name="text">
                    <string>Level</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="ConfigOutputAudio" name="audioConfig" native="true"/>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ConfigOutputAudio</class>
   <extends>QWidget</extends>
   <header>../src/configurations/configoutputaudio.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>useCustomBufferSizeBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>bufferSizeBox</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>447</x>
     <y>631</y>
    </hint>
    <hint type="destinationlabel">
     <x>859</x>
     <y>631</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useAutomaticBitrateBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>minBitrateLabel</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>447</x>
     <y>631</y>
    </hint>
    <hint type="destinationlabel">
     <x>859</x>
     <y>631</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useAutomaticBitrateBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>minBitrateBox</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>447</x>
     <y>631</y>
    </hint>
    <hint type="destinationlabel">
     <x>859</x>
     <y>631</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>advancedModeBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>widget_2</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>578</y>
    </hint>
    <hint type="destinationlabel">
     <x>51</x>
     <y>634</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
